machine-learning-scikit-learn.ipynb

```diff
@@ -0,0 +1,333 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Scikit-learn + Dask"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Scikit-learn is a popular machine learning library, and implements the minimal API required for training machine learning models. Their library encompasses a wide variety of problem formulations, ranging from simple linear models to more complex clustering or decomposition formulations. Scikit-learn is hindered by computation (what machine library isn't?). This has even gotten a page in their user guide, \"[Strategies to scale computationally: bigger data][strat]\".\n",
+    "\n",
+    "Using scikit-learn estimators with Dask-ML helps scale to large datasets. All the computation remains on scikit-learn's shoulders but the data management is handled by Dask. This allows scaling to large datasets distributed across many machines, or to datasets that do not fit in memory.\n",
+    "\n",
+    "This example wraps a scikit-learn estimator with a Dask-ML estimator suited for incremental learning. This Dask-ML estimator manages feeding data to Scikit-learn, and leaves the computation to Scikit-learn. This example will not cover the other algorithms inside of Dask-ML that are well-suited for Dask's distributed architecture.\n",
+    "\n",
+    "This example will show\n",
+    "\n",
+    "* wrapping a scikit-learn estimator that implement `partial_fit` with `dask_ml`\n",
+    "* training, predicting, and scoring on this wrapped classifier\n",
+    "* integration with other parts of sklearn (e.g., with GridSearchCV)\n",
+    "\n",
+    "<img src=\"http://scikit-learn.org/stable/_static/scikit-learn-logo-small.png\">\n",
+    "<img src=\"https://www.continuum.io/sites/default/files/dask_stacked.png\" width=\"100px\">\n",
+    "\n",
+    "[strat]:http://scikit-learn.org/stable/modules/scaling_strategies.html\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Data creation\n",
+    "We will create many synthetic data. We are more interested in showing the plumbing this data can flow through than cool insights we can glean from the data.\n",
+    "\n",
+    "Let's create some synthetic data that's not too large (so it runs in a reasonable time for you) and but is realistically large. We have 100,000 examples and 100 features in this dataset we create."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sklearn.datasets\n",
+    "import dask.array as da\n",
+    "import numpy as np\n",
+    "n, d = int(100e3), 100\n",
+    "\n",
+    "X = da.random.normal(size=(n, d), chunks=n // 10)\n",
```

Comment:
> dask/dask-ml#255 would cover this right?

It would help. I've gotten rid of `w_star` – I think I was using it for ground truth, but I don't actually use it later.